raft/raftpb/raft.proto

```diff
@@ -43,6 +43,7 @@ enum MessageType {
 	MsgHeartbeat     = 8;
 	MsgHeartbeatResp = 9;
 	MsgUnreachable   = 10;
+	MsgSnapStatus    = 11;
```

Comment:
Should this be MsgSnapResp instead? In the success case, an application may want to wait to report the status until the follower has finished processing the snapshot, in which case the MsgSnapResp may actually be sent over the wire.  (or maybe the successful wire message for a snapshot is just an MsgAppResp).

Whether we allow it over the wire or not, the new message type should be added to either IsLocalMsg or IsResponseMsg in util.go.